tracer/src/Datadog.Trace/ExtensionMethods/SpanExtensions.cs

```diff
@@ -38,31 +38,6 @@ public static void AddTagsFromDbCommand(this ISpan span, IDbCommand command)
             }
         }
 
-        /// <summary>
-        /// Add the StackTrace and other exception metadata to the span
-        /// </summary>
-        /// <param name="span">The span to add the exception to.</param>
-        /// <param name="exception">The exception.</param>
-        public static void SetException(this ISpan span, Exception exception)
```

Comment:
Unused by us, but is it useful for users? 🤔  I'm still inclined to remove and wait for the screams. Does that make me a bad person?

Reply:
Not to fear! This method was previously defined on `Span` and is now defined on `ISpan`. It was only defined as an extension method in the middle of this PR 😄